common/src/main/java/org/opensearch/ml/common/model/TextEmbeddingModelConfig.java

```diff
@@ -184,7 +184,7 @@ public String getName() {
 
         public static PoolingMode from(String value) {
             try {
-                return PoolingMode.valueOf(value);
+                return PoolingMode.valueOf(value.toUpperCase(Locale.ROOT));
```

Comment:
Not this line.

 do we need to change `weightedmean` value?

Reply:
I prefer to change it in a separate PR. That change takes some time to test.